framework/test/mocha/unit/modules/chain/chain.js

```diff
@@ -183,10 +183,10 @@ describe('Chain', () => {
 					id: '123',
 					height: '1',
 				});
-				const response = await chain.actions.getHighestCommonBlockId({
+				const response = await chain.actions.getHighestCommonBlock({
 					params: { ids: ['1', '2'] },
 				});
-				expect(response).to.equal('123');
+				expect(response).to.deep.equal({ id: '123', height: '1' });
```

Comment:
```suggestion
				expect(response).to.eql({ id: '123', height: '1' });
```
Probably it's the same?

Reply:
It's the same in Jest, but in Mocha if you do just `equal` it will perform reference comparison and hence fail.